LICENSES/simplified_bsd.txt

```diff
@@ -0,0 +1,7 @@
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```

Comment:
The filename should be `BSD-2-Clause.txt`.

Reply:
ah ok, I had based it on this (earlier) PR 😅 : https://github.com/ansible-collections/community.general/pull/4759/files#diff-cf536a04b1dc7d11785cb3d01164cbada12d6613c885f58d808826274275a883